### PR TITLE
Add offset value for Eternal Teleport Crystal in the ItemsKeptOnDeath…

### PR DESCRIPTION
**Pull request by thomaschow**
… plugin

### Description

- Addresses https://github.com/runelite/runelite/issues/11187

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/itemskeptondeath/FixedPriceItem.java [hunk 1]
```diff
@@ -219,6 +219,9 @@
 	VERACS_FLAIL_100(ItemID.VERACS_FLAIL_100, 10000, ItemID.VERACS_FLAIL_0),
 
 	AVERNIC_DEFENDER(ItemID.AVERNIC_DEFENDER, 4040000),
+
+	ETERNAL_TELEPORT_CRYSTAL(ItemID.ETERNAL_TELEPORT_CRYSTAL, 1),
```
Comment (thomaschow) [inline runelite-client/src/main/java/net/runelite/client/plugins/itemskeptondeath/FixedPriceItem.java:223]: Would like some guidance on how to determine the exact offset value.

Comment (thomaschow) [reply runelite-client/src/main/java/net/runelite/client/plugins/itemskeptondeath/FixedPriceItem.java:223]: Value `500` is taken from osrs wiki: https://oldschool.runescape.wiki/w/Eternal_teleport_crystal

**Comment by Hydrox6** [comment]
It's usually a bad idea to make a PR that you know is definitely wrong.

**Comment by thomaschow** [comment]
I'll close this and verify the values, will re-open later.

**Pull request closed without merging**